### ISSUE TITLE
Design drafts

### ISSUE THREAD

**Issue by ClairePUK** [ParkisonsUK-fuse/what3things#97]
From design team: Please find attached first drafts – nothing fixed – all changeable – but a starting point. This is just a flavour of the design theme we're exploring, we've tried to keep it simple with some variation.

[3Things_Artwork001.pdf](https://github.com/ParkisonsUK-fuse/what3things/files/946903/3Things_Artwork001.pdf)

**Comment by andrewMacmurray** [comment]
@ClairePUK hmm I think we'll need to have a chat about these

**Comment by ClairePUK** [comment]
@andrewMacmurray Probably worth speaking with the design team during their session on Tuesday. Shall we plan a Google Hangouts chat once I know what times they will be in the lab from?

**Comment by andrewMacmurray** [comment]
I think that would be good, the sooner the better. Are going to have a chance to chat with them before? It might be good to run through with you a couple of my concerns with implementing them technically

**Comment by ClairePUK** [comment]
Yeah, we'll try and get the call in before the session starts, let's aim for 10am. I'm actually on leave on Monday, and working from home today, so Tuesday will be the first chance I get to speak to them.

**Comment by ivanmauricio** [comment]
[Wondering what happens next.pdf](https://github.com/ParkisonsUK-fuse/what3things/files/961734/Wondering.what.happens.next.pdf)

**Comment by ClairePUK** [comment]
[RE__Design_assests.zip](https://github.com/ParkisonsUK-fuse/what3things/files/964354/RE__Design_assests.zip)

The squished speech bubble on the results page is just speech bubble no 1 squished. Is this something you can do or will you need a separate version?

**Comment by ClairePUK** [comment]
[LICENSE.txt](https://github.com/ParkisonsUK-fuse/what3things/files/964637/LICENSE.txt)
